project.clj

```diff
@@ -17,6 +17,9 @@
   :mranderson {:project-prefix "mranderson.inlined"}
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.8.0"]
                                   [leiningen-core "2.9.1"]]}
+             :eastwood {:dependencies [[org.clojure/clojure "1.10.3"]]
+                        :plugins      [[jonase/eastwood "0.9.1"]]
```

Comment:
Starting from 0.9.0, Eastwood also lints for reflection warnings which helped me develop this PR.

You can learn more in https://github.com/jonase/eastwood/#reflection